dist/constant/Makefile.PL

```diff
@@ -0,0 +1,30 @@
+use 5.006;
+use strict;
+use ExtUtils::MakeMaker;
+
+my %prereq = (
+    "Test::More" => 0,
+);
+
+WriteMakefile(
+    NAME            => 'constant',
+    LICENSE         => 'perl',
+    AUTHOR          => 'Sebastien Aperghis-Tramoni <sebastien@aperghis.net>',
+    VERSION_FROM    => 'lib/constant.pm',
+    ABSTRACT_FROM   => 'lib/constant.pm',
+    INSTALLDIRS     => ( $] < 5.011 ? 'perl' : 'site' ),
+    PL_FILES        => {},
+    PREREQ_PM       => \%prereq,
+    META_MERGE          => {
+        resources       => {
+            repository  => 'https://github.com/Perl/perl5.git',
+            license     => 'http://dev.perl.org/licenses/',
+            homepage    => 'https://metacpan.org/module/constant',
+            irc         => 'irc://irc.perl.org/#p5p',
+            mailinglist => 'http://lists.perl.org/list/perl5-porters.html',
+            bugtracker  => 'https://github.com/Perl/perl5/Issues',
```

Comment:
Lower case "I" in issues, this one 404s.

The rest look fine.

It's possible to expand the resources key out by switching to meta-spec 2, (see `cpan/Scalar-List-Utils/Makefile.PL` for example) but I'm mostly trying to pick up obvious errors.  Refactoring to make them all meta-spec 2 seems out of scope for me.

Reply:
Fixed. Re: meta-spec 2, I think in the future we should look at making these Makefile.PL's more consistent, but I agree it is not in scope now.